model/work.py

```diff
@@ -1173,11 +1166,13 @@ def active_license_pool(self):
                     # We have an unlimited source for this book.
                     # There's no need to keep looking.
                     break
+            elif p.self_hosted:
+                active_license_pool = p
+                break
```

Comment:
It will almost never matter but I don't think `break` is appropriate here. If there is a self-hosted non-open-access edition of a book, but also an open-access edition, then the open-access edition should win because it comes with fewer strings attached.

Reply:
Removed the `break` statement